src/runtime_ccall.cpp

```diff
@@ -146,21 +146,27 @@ void *jl_get_library(const char *f_lib)
 #endif
     if (f_lib == NULL)
         return jl_RTLD_DEFAULT_handle;
-    hnd = libMap[f_lib];
+    JL_LOCK_NOGC(&libmap_lock);
```

Comment:
that's why I said, 'on windows': https://github.com/libuv/libuv/blob/v1.x/src/win/thread.c#L173